cli/validate_inputs.go

```diff
@@ -53,7 +53,7 @@ func validateTerragruntInputs(terragruntOptions *options.TerragruntOptions, work
 		}
 		terragruntOptions.Logger.Warn("")
 	} else {
-		terragruntOptions.Logger.Info("All variables passed in by terragrunt are in use.")
+		terragruntOptions.Logger.Info(fmt.Sprintf("All variables passed in by terragrunt are in use. Strict mode enabled: %t", terragruntOptions.Strict))
```

Comment:
NIT: strict mode info should be debug logs, not info.

Reply:
Done in 139b820